pysilcam/silcam_auv.py

```diff
@@ -0,0 +1,274 @@
+# -*- coding: utf-8 -*-
+import os
+import sys
+from datetime import datetime, timedelta
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.dates as mdates
+
+pysiclam_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.prepend(pysiclam_dir)
+
+import pysilcam.postprocess as scpp
+from pysilcam.config import PySilcamSettings
```

Comment:
E402: module level import not at top of file  <!-- Automatically posted by Lintly -->